Getting internal server error on test case when run locally

When running main locally on macos I get test failures

```
$ pnpm run test
...snip...
 FAIL  test.ts > Testset: ebolazaire_1563 > should validate query from query1.query.json
AssertionError: {"error":{"type":"about:blank","title":"Internal Server Error","status":500,"detail":"Could not connect to silo: class java.net.ConnectException null"},"info":{"dataVersion":null,"requestId":"43df4c90-93a0-41ae-8aae-73c49c047b19","requestInfo":"Ebola Zaire on localhost at 2024-09-05T11:54:24.819757632","reportTo":"Please report to https://github.com/GenSpectrum/LAPIS/issues in case you encounter any unexpected issues. Please include the request ID and the requestInfo in your report."}}: expected 500 to equal 200
```

<img width="2554" alt="iTerm2 2024-09-05 13 55 21" src="https://github.com/user-attachments/assets/899440c8-095a-404e-a4e9-26c864a6db1f">

--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -13,6 +13,13 @@ services:
               source: ${TESTSET_DATA_FOLDER}/reference_genomes.json
               target: /workspace/reference_genomes.json
               read_only: true
+        healthcheck: # TODO(https://github.com/GenSpectrum/LAPIS-SILO-e2e/issues/32) Remove at some point
+          test: ["CMD-SHELL",
+            "wget -qO- http://localhost:8080/sample/aggregated | grep -o '\"count\":[0-9]*' | sed 's/[^0-9]//g' | xargs test 0 -ne || exit 1"]
+          interval: 2s
+          timeout: 5s
+          retries: 10
+
 
     silo:
         image: ghcr.io/genspectrum/lapis-silo:${SILO_TAG}

--- a/test.ts
+++ b/test.ts
@@ -56,7 +56,7 @@ testsets.map((testSuite) => {
     function dockerComposeUp() {
         console.log(`Starting Docker Compose for ${testSuite.path}...`);
 
-        const dockerComposeUpCommand = `${dockerComposeEnv} docker compose --project-name ${testName} --progress=plain up --no-recreate --detach --wait && sleep 4`;
+        const dockerComposeUpCommand = `${dockerComposeEnv} docker compose --project-name ${testName} --progress=plain up --no-recreate --detach --wait`;
 
         console.log(dockerComposeUpCommand);
         execSync(dockerComposeUpCommand, { stdio: 'inherit' });
@@ -65,9 +65,7 @@ testsets.map((testSuite) => {
     function dockerComposeDown() {
         console.log(`Stopping Docker Compose for ${testSuite.path}...`);
 
-        // Add the sleep 1 because it takes some time until the port is available again.
-        // This might be relevant when running vitest with auto restart
-        const dockerComposeDownCommand = `docker compose -p ${testName} --progress=plain down && sleep 1`;
+        const dockerComposeDownCommand = `docker compose -p ${testName} --progress=plain down`;
 
         console.log(dockerComposeDownCommand);
         execSync(dockerComposeDownCommand, { stdio: 'inherit' });

